### PR TITLE
Updated existing PDF.co connector with version 2.0 with all missing functionality from pdf.co.  

### PR DESCRIPTION
**Pull request by hiren-byte**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/PDFco/apiDefinition.swagger.json [hunk 1]
```diff
@@ -649,52 +880,7791 @@
...
+                  "format": "int32",
+                  "description": "",
+                  "title": "Page Count",
+                  "x-ms-visibility": "advanced"
+                },
+                "error": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "Error",
+                  "enum": [
```
Comment (vmanoharas) [inline certified-connectors/PDFco/apiDefinition.swagger.json:1548]: Please change "true" to true and "false" false, please see the below swagger validation error. 

##[error]The type of the enum value is 'String' but it should be 'Boolean' as specified in the schema. ValuePath: /paths/~1v1~1barcode~1generate/post/responses/200/schema/properties/error/enum

Please refer this schema link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json#L1002

Please fix all responses in the swagger file.

File: certified-connectors/PDFco/apiDefinition.swagger.json [hunk 2]
```diff
@@ -38,8 +26,8 @@
     "/v1/pdf/convert/from/html": {
       "post": {
         "responses": {
-          "200": {
-            "description": "OK, success",
+          "default": {
```
Comment (vmanoharas) [inline certified-connectors/PDFco/apiDefinition.swagger.json]: it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all responses in the swagger file.

**Review by vmanoharas — COMMENTED** [review]
Hello @hiren-byte, 

Please resolve review comments, additionally, please check swagger validation errors and fix them duly.

File: certified-connectors/PDFco/apiDefinition.swagger.json (hunk 1 above)
Comment (hiren-byte) [reply certified-connectors/PDFco/apiDefinition.swagger.json:1548]: @vmanoharas , I have fixed this.

File: certified-connectors/PDFco/apiDefinition.swagger.json (hunk 2 above)
Comment (hiren-byte) [reply certified-connectors/PDFco/apiDefinition.swagger.json]: @vmanoharas , I have modified response codes

**Comment by hiren-byte** [comment]
@vmanoharas , Thank you for review! I have updated `apiDefinition.swagger.json` file to fix the issues.

**Review by vmanoharas — COMMENTED** [review]
Hello @hiren-byte, 

Thank you for fixing previous comments, however, can you please look into below swagger validation and confirm that Existing flows may not affect? please


##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFFromEmail/parameter/body/schema/embedAttachments/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFFromEmail/parameter/body/schema/convertAttachments/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFFromEmail/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowAccessibilitySupport/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowAssemblyDocument/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowPrintDocument/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowFillForms/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowModifyDocument/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowContentExtraction/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/allowModifyAnnotations/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFAddSecurity/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFSecurityRemove/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/PDFFromXLSXLSX/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/XLStoCSV/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/XLStoJSON/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/XLStoHTML/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/XLStoTXT/parameter/body/schema/async/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/XLStoXML/parameter/body/schema/async/enum Source: NewSwagger

**Comment by hiren-byte** [comment]
@vmanoharas, yes, I confirm that this won't affect existing flows. Also, these validation errors are in the new modules which are not in public domain.

**Review by vmanoharas — APPROVED** [review]
Hello @hiren-byte 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**